data/mods/Aftershock/snippets/effect_on_condition.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "type": "snippet",
+    "category": "AFS_NIGHT_MESSAGES_1",
+    "text": [
+      "A mountainside in the distance is littered with erratic flashes of light that suggest extended combat.  It's too far to tell any other details.",
+      "You hear a sudden CRACK of ice breaking and then the low roar of what must be an avalanche just outside your line of sight.",
+      "Winged shadows cross the ground but when you look into the sky you can't seem to see any movement or shapes at all.",
+      "Catch a falling star and put it in your pocket, watch it blaze on through the night sky.  It's a peaceful sign despite all the chaos here below.",
+      "Some craft swiftly hovers across the night sky, searchlights flaring out from it's belly in figure eight patterns.  It circles in an ever larger pattern until it pauses and weapons fire lashes from it to the ground like lightning.  A full five seconds of broadsides to the ground, then only the searchlights pointed where the shots seemed to land, then even that light cuts off and it might as well be invisible to you.",
```

Comment:
```suggestion
      "Winged shadows cross the ground, but when you look into the sky you can't seem to see any movement or shapes at all.",
      "Catch a falling star and put it in your pocket, watch it blaze on through the night sky.  It's a peaceful sign despite all the chaos here below.",
      "Some craft swiftly hovers across the night sky, searchlights flaring out from its belly in figure-eight patterns.  It circles in an ever larger pattern until it pauses and weapons fire lashes from it to the ground like lightning.  A full five seconds of broadsides to the ground, then only the searchlights pointed where the shots seemed to land, then those lights cut off and it might as well be invisible to you.",
```